### PR TITLE
README

### PR DESCRIPTION
**Pull request by ykurata**
This is the first version of README. I think there are lots of missing parts, please comment me!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+    **Frontend (Client)**
+
+    * Go to client directory, and run `npm install`.
+
+    * To start client, run `npm start`.
+
+    **Backend (Server)**
+        
+    * Got to server directory, and run `npm install`.
```
Comment (Kartihan-sri) [inline README.md]: Should be "Go"

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+    * To start server, run `npm start`.
+
+* Ensure that you have MongoDB installed globally on your system.
+
+    *  Open a `Command` or `Terminal` and run `mongod (or sudo mongod)` to start
```
Comment (Kartihan-sri) [inline README.md]: Don't need "a" here also should it be "Command" or "Command Prompt"?

**Review by Kartihan-sri — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+    
+* MongoDB 
+    * The project uses MongoDB as a database. Installation tutorial is here [here].
+
+[here]: https://docs.mongodb.com/manual/installation/
+
+
+## Getting Started 
+
+* To get started, clone the repo to your local machime.
```
Comment (romaldowoho) [inline README.md]: Machine

File: README.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+
+    ```https://github.com/hatchways/team-hazelnut.git```
+
+* Install all dependencies on the both fron end and back end.
```
Comment (romaldowoho) [inline README.md]: front

File: README.md (hunk 1 above)
Comment (ykurata) [reply README.md]: so many typos...

**Pull request merged**